File: C/Iniciante/1011.c
#include <stdio.h>
#include <math.h>
 
int main() {
 
    double valor, valorFinal;
    
    printf("");
    scanf("%lf", &valor);
    
    double pi = 3.14159;
    
    valorFinal = (4/3.0) * pi * pow(valor, 3);
    
    printf("VOLUME = %.3lf\n", valorFinal);
 
    return 0;
}

File: C#/Iniciante/1011.cs
using System; 

class URI {

    static void Main(string[] args) { 
       double valorRaio = double.Parse(Console.ReadLine());
       double pi = 3.14159;
       double resultadoFinal = (4/3.0) * pi * Math.Pow(valorRaio, 3);
       
       Console.WriteLine($"VOLUME = {resultadoFinal:F3}");
    }

}